### DISCUSSION TITLE
Future of Branding - DataSaturday v SQL Saturday v xxx

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#113] [Ideas]
Now that Redgate has purchased the PASS assets, the fate of the SQLSaturday URL and trademark is settled. I am opening this debate to discuss how this might move forward.

Disclosure: I work for Redgate Software

There is an issue, #17 , already open on this, but  a discussion is likely more appropriate.

The SQLSaturday brand is near and dear to my heart. It is also widely recognized and recognized, by speakers, attendees, and sponsors. That is important for growth, sustainability, and stability.

I think there are 3 ways forward here:
1. Merge - Have these two, and maybe others, point to one site/set of tools
2. Hierarchy. I could see DataSaturdays.com as a top brand, that shows SQLSaturdays, PowerBI Saturdays, etc. shown on its site,  providing links, tools, etc. to everyone
3. Something else.

**Comment by rabryst** [comment]
Option 2 works for me. Once PASS went away it was clear that they had been under the impression they *were* the community, which is demonstrably wrong. This also allows Redgate to bring back the SQLSaturday brand as a support of a community endeavour as opposed to taking it over entirely.

**Comment by amtwo** [comment]
> The fate of the SQLSaturday URL and trademark is settled

I think it would be helpful to provide some clarity/transparency as to what Redgate is planning to do with the brand/trademark/etc.

Some random thoughts/questions--some of these overlap, but it's honestly a bit stream-of-thought:
* Is Redgate looking to fully turn those assets over to an open-source project? This project?
* Is Redgate interested in "owning" the project? Participating in a separate project? Spinning out a Foundation to own the project?
* What would Redgate's involvement look like? What are they looking to invest into SQLSaturday? Cash? Development resources? Marketing/branding resources? 
* On the other side of their investment--Does Redgate have guidelines or objectives or other goals for a return on investment?

**Reply by way0utwest** [reply]
There are no plans as of yet. This happened too quickly.

My proposal for them is to put SQLSat into a foundation and have it support and inspire events. No project for tools, but coordinate/support/use this project.

Involvement - TBD. Nothing is decided. Looking for input here.

ROI - Nothing direct here. We do want to run the Summit event in the future and be associated with things.

**Comment by way0utwest** [comment]
FWIW: related discussion here: https://www.sqlservercentral.com/forums/topic/data-saturdays-and-sqlsaturday
